src/index.html

```diff
@@ -1,6 +1,19 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="D:\projects\layout_search-bar-airbnb\src\fonts"
+    />
+    <link
+      rel="preconnect"
+      href="D:\projects\layout_search-bar-airbnb\src\fonts"
+      crossorigin="anonymous"
+    />
+    <link
+      href="D:\projects\layout_search-bar-airbnb\src\fonts"
```

Comment:
The `href` attribute for the stylesheet link is pointing to a local file path. Ensure that the path is correct and accessible from the HTML file.